CMakePresets.json

```diff
@@ -226,6 +226,37 @@
         "CCCL_ENABLE_EXAMPLES": true,
         "CCCL_ENABLE_TESTING": true
       }
+    },
+    {
+      "name": "all-dev",
+      "inherits": "base",
+      "cacheVariables": {
+        "CCCL_ENABLE_LIBCUDACXX": true,
+        "CCCL_ENABLE_CUB": true,
+        "CCCL_ENABLE_THRUST": true,
+        "CCCL_ENABLE_TESTING": true,
+        "CCCL_ENABLE_EXAMPLES": true,
+        "LIBCUDACXX_ENABLE_LIBCUDACXX_TESTS": true,
+        "CUB_ENABLE_TESTING": true,
+        "CUB_ENABLE_EXAMPLES": true,
+        "CUB_SEPARATE_CATCH2": true,
+        "CUB_IGNORE_DEPRECATED_CPP_DIALECT": true,
+        "CUB_ENABLE_DIALECT_CPP11": true,
+        "CUB_ENABLE_DIALECT_CPP14": true,
+        "CUB_ENABLE_DIALECT_CPP17": true,
+        "CUB_ENABLE_DIALECT_CPP20": true,
+        "THRUST_ENABLE_MULTICONFIG": true,
+        "THRUST_MULTICONFIG_WORKLOAD": "LARGE",
+        "THRUST_IGNORE_DEPRECATED_CPP_DIALECT": true,
+        "THRUST_MULTICONFIG_ENABLE_DIALECT_CPP11": true,
+        "THRUST_MULTICONFIG_ENABLE_DIALECT_CPP14": true,
+        "THRUST_MULTICONFIG_ENABLE_DIALECT_CPP17": true,
+        "THRUST_MULTICONFIG_ENABLE_DIALECT_CPP20": true,
+        "THRUST_MULTICONFIG_ENABLE_SYSTEM_CPP": true,
+        "THRUST_MULTICONFIG_ENABLE_SYSTEM_CUDA": true,
+        "THRUST_MULTICONFIG_ENABLE_SYSTEM_OMP": true,
+        "THRUST_MULTICONFIG_ENABLE_SYSTEM_TBB": true
```

Comment:
👀

Reply:
Thrust's build system is a *beast*: https://github.com/NVIDIA/cccl/blob/main/thrust/cmake/ThrustMultiConfig.cmake#L40-L70 😄